src/Core/Models/Mail/OrganizationUserInitInvitedViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Bit.Core.Models.Mail;
+
+public class OrganizationUserInitInvitedViewModel : BaseMailModel
```

Comment:
In this case I do think it's worth re-using or subclassing the `OrganizationUserInvitedViewModel`. The only difference is the `Url` property, everything else is the same including the mail template.